terraso_backend/apps/soil_id/models/project_soil_settings.py

```diff
@@ -0,0 +1,57 @@
+from django.db import models
+
+from apps.core.models.commons import BaseModel
+from apps.project_management.models.projects import Project
+from apps.soil_id.models.depth_interval import BaseDepthInterval
+
+
+class ProjectSoilSettings(BaseModel):
+    project = models.OneToOneField(Project, on_delete=models.CASCADE, related_name="soil_settings")
+
+    class MeasurementUnit(models.TextChoices):
+        IMPERIAL = "IMPERIAL"
+        METRIC = "METRIC"
```

Comment:
question: how does `MeasurementUnit.choices` get defined? This looks like a neat pattern; perhaps we could reuse it elsewhere to reduce duplication.

Reply:
Yes — please do.